setup.cfg

```diff
@@ -72,6 +72,7 @@ install_requires =
     starlette
     typing-extensions>=4.0
     backports.cached-property;python_version<"3.8"
+    exceptiongroup;python_version <= "3.11"
```

Comment:
This will be the default package for python 3.11. This would actually provide better exception traceback than rich traceback.
<img width="1059" alt="Screenshot 2022-07-05 at 00 35 59" src="https://user-images.githubusercontent.com/29749331/177274951-5b692f8d-8048-404b-8248-0f24f0ff88ef.png">

Reply:
rich traceback haven't supported this yet. We could allow user to enable rich.traceback if then want to. But I think for now exceptiongroup is more verbose.

Reply:
We should move back to rich.traceback once https://github.com/Textualize/rich/issues/1859 supports it :)